vtex-apps/add-to-cart-button
# Add a new onClickEventPropagation prop

#### What problem is this solving?

This should enable users to turn off the default `event.stopPropagation()` called inside of `handleAddToCart` function. This is useful if they want to wrap the button with any `*-trigger` blocks, such as `modal-trigger`.

#### How to test it?

Go to [Workspace](https://victormiranda--storecomponents.myvtex.com/apparel---accessories/) and add a product to the cart while still on a search page. You should see a `modal-layout` with the product's info.

#### Related to / Depends on

This should solve the issue reported by https://github.com/vtex-apps/store-discussion/issues/485 and also renders https://github.com/vtex-apps/modal-layout/pull/40 unnecessary.

#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/1hAxQTH0HEWS3L0oRF/giphy.gif)
